src/borg/testsuite/archiver/checks.py

```diff
@@ -46,249 +46,243 @@ def cmd_raises_unknown_feature(archiver, args):
 
 def test_repository_swap_detection(archivers, request):
     archiver = request.getfixturevalue(archivers)
-    repo_location, repo_path, input_path = archiver.repository_location, archiver.repository_path, archiver.input_path
-    create_test_files(input_path)
+    create_test_files(archiver.input_path)
     os.environ["BORG_PASSPHRASE"] = "passphrase"
-    cmd(archiver, f"--repo={repo_location}", "rcreate", RK_ENCRYPTION)
-    repository_id = _extract_repository_id(repo_path)
-    cmd(archiver, f"--repo={repo_location}", "create", "test", "input")
-    shutil.rmtree(repo_path)
-    cmd(archiver, f"--repo={repo_location}", "rcreate", "--encryption=none")
-    _set_repository_id(repo_path, repository_id)
-    assert repository_id == _extract_repository_id(repo_path)
+    cmd(archiver, "rcreate", RK_ENCRYPTION)
+    repository_id = _extract_repository_id(archiver.repository_path)
+    cmd(archiver, "create", "test", "input")
+    shutil.rmtree(archiver.repository_path)
+    cmd(archiver, "rcreate", "--encryption=none")
+    _set_repository_id(archiver.repository_path, repository_id)
+    assert repository_id == _extract_repository_id(archiver.repository_path)
     if archiver.FORK_DEFAULT:
-        cmd(archiver, f"--repo={repo_location}", "create", "test.2", "input", exit_code=EXIT_ERROR)
+        cmd(archiver, "create", "test.2", "input", exit_code=EXIT_ERROR)
     else:
         with pytest.raises(Cache.EncryptionMethodMismatch):
-            cmd(archiver, f"--repo={repo_location}", "create", "test.2", "input")
+            cmd(archiver, "create", "test.2", "input")
 
 
 def test_repository_swap_detection2(archivers, request):
     archiver = request.getfixturevalue(archivers)
-    repo_location, repo_path, input_path = archiver.repository_location, archiver.repository_path, archiver.input_path
-    create_test_files(input_path)
-    cmd(archiver, f"--repo={repo_location}_unencrypted", "rcreate", "--encryption=none")
```

Comment:
There were only a handful of instances where the first argument was not **exactly**`f"--repo={repo_location}`. This file had most of those instances, so I updated `archiver.repository_location` to mimic the behavior.